Amadeus-frontend/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:22-alpine
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+RUN npm run build 
+RUN npm install -g serve
+
+EXPOSE 3000
```

Comment:
you can use a second stage, run to only run the application with the output of the build/install, that way you don't expose any code 

https://docs.docker.com/build/building/multi-stage/